reference/6/Microsoft.PowerShell.Utility/Get-Member.md

```diff
@@ -145,10 +145,19 @@ The results show that only process objects (**System.Diagnostics.Process**) and
 
 ### Example 6: Get members for an array
 ```
-PS C:\> $A = Get-Member - InputObject @(1)
+PS C:\> $A = @(1)
```

Comment:
If you are going to include the PowerShell prompt in examples, please use "PS>" for the prompt. See https://github.com/PowerShell/PowerShell-Docs/blob/staging/contributing/5-FORMATTING-CODE.md#simple-illustration-example

Please make this change to each version of the file.